### PR TITLE
add r-prim

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

File: recipes/r-prim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  license: GPL-2 | GPL-3
+  summary: Patient Rule Induction Method (PRIM) for bump hunting in high-dimensional data.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - FernandezMathieu
```
Comment (dbast) [inline recipes/r-prim/meta.yaml:53]:
```suggestion
    - conda-forge/
    - FernandezMathieu
```

**Comment by dbast** [comment]
Packages using r-rgl need more setup code to work. There should be some other packages which help as an example.

**Comment by FernandezMathieu** [comment]
Look like adding yum requirements fix the linux build but not the osx. 
Any idea what to add to build successfully ? @dbast

**Comment by dbast** [comment]
As this is a noarch package it will be built on linux after feedstock conversion...

**Pull request merged**